draft-ietf-teep-architecture.md

```diff
@@ -619,33 +619,30 @@ complex than Cases 2 and 3.
 ## Entity Relations
 
 This architecture leverages asymmetric cryptography to
-authenticate a device to a TAM. Additionally, a TEE
-in a device authenticates a TAM and TA signer. The
+authenticate a device to a TAM. Additionally, a TEE Agent
+in a device authenticates a TAM. The
 provisioning of Trust Anchors to a device may be different from
 one use case to the other. A Device Administrator may want to
 have the capability to control what TAs are allowed.
-A device manufacturer enables verification of the TA signers
-and TAM providers; it may embed a list of default Trust Anchors
-that the signer of an allowed TA's signer certificate should
-chain to. A Device Administrator may choose to accept a subset
-of the allowed TAs via consent or action of downloading.
+A device manufacturer enables verification of the TAM providers and TA binary signers; 
+it may embed a list of default Trust Anchors into the TEEP Agent
+and TEE for TAM trust verification and TA signer verification. 
 
 ~~~~
  (App Developer)    (App Store)    (TAM)     (Device with TEE)  (CAs)
-        |                                            |
-        |                               --> (Embedded TEE cert) <--
-        |                                            |
-        | <------------------------------  Get an app cert ----- |
-        |                           | <--  Get a TAM cert ------ |
-        |
+        |               |           |                |            |
+        |               |           |      (Embedded TEE cert) <--|
+        |               |           |                |            |
+        | <--- Get an app cert -----------------------------------|
+        |               |           | <-- Get a TAM cert ---------|
+        |               |           |                |            |
 1. Build two apps:
-   Untrusted Application
-       TA
-        |
-        |
-   Untrusted Application -- 2a. --> | ----- 3. Install -------> |
-       TA ----------------- 2b. Supply ------> | 4. Messaging-->|
-        |                           |          |                |
+     UTA, TA
+        |               |           |                |            |
+        |               |           |                |            |
+       UTA -- 2a. --->  | --- 3. Install ----------> |            |
+       TA --- 2b. Supply ---------> | 4. Messaging-->|            |
+        |               |           |                |            |
 ~~~~
 {: #experience title="Developer Experience"}
 
```

Comment:
Line 653-654: The same developer building both the Untrusted App and TA is just one example.
So "shows an example where the same developer builds two applications:"

(Above also fixes the typo of "an TA")